src/sites_toolbox.py

```diff
@@ -28,6 +30,36 @@ def get_user_details(session, email):
     ]
     return user_sites, user_site_group, user_site_count
 
+# get details for one site
+
+def get_site_details(session, site_selection):
+  """Get the site details for one site"""
+  site_uuid = get_site_by_uuid(session=session, site_uuid=site_selection)
+  site_details = {"site_uuid": str(site_uuid.site_uuid), 
+                  "client_site_id": str(site_uuid.client_site_id),
+                  "client_site_name": str(site_uuid.client_site_name),
+                  "site_group_name": str(site_uuid.site_groups[0].site_group_name),
+                  "latitude": str(site_uuid.latitude),
+                  "longitude": str(site_uuid.longitude),
+                  "DNO": str(site_uuid.dno),
+                  "GSP": str(site_uuid.gsp),
+                  "tilt": str(site_uuid.tilt),
+                  "orientation": str(site_uuid.orientation),
+                  "capacity": (f'{site_uuid.capacity_kw} kw'),
+                  "date_added": (site_uuid.created_utc.strftime("%Y-%m-%d"))}
+  return site_details
+
+def select_site_id(dbsession, query_method):
+        if query_method == "site_uuid":
```

Comment:
could you add a function description to say what this does

Reply:
worth perhaps still adding a function descirption, but just do this in a seperate PR